lib/anyplayer/selector.rb

```diff
@@ -1,6 +1,7 @@
-require "rbconfig"
+require 'rbconfig'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.